include/grpc/impl/codegen/port_platform.h

```diff
@@ -217,12 +213,7 @@
 #define GPR_CPU_POSIX 1
 #define GPR_POSIX_CRASH_HANDLER 1
 #endif
-#ifdef __has_feature
-#if __has_feature(cxx_thread_local)
-#define GPR_STDCPP_TLS 1
-#endif
-#endif
-#ifndef GPR_STDCPP_TLS
+#if !(defined(__has_feature) && __has_feature(cxx_thread_local))
```

Comment:
one open question: I could move this feature detection directly into tls.h and be rid of GPR_*_TLS altogether. Do folks have a preference?

Reply:
Keep it here... we blanket replace this file in some other environments with a hand-rolled one where we don't necessarily want feature detection (this case wouldn't hurt, but consistency matters)